src/app/lib/presentation/configuration/environment_configuration.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+
+/// This entity contains environment specific configurations.
+class EnvironmentConfiguration {
+  final bool isConsoleLoggingEnabled;
+  final bool isFileLoggingEnabled;
+  final String dadJokesBaseUrl;
+
+  EnvironmentConfiguration(DotEnv dotEnv)
+      : isConsoleLoggingEnabled =
+            dotEnv.env["IS_CONSOLE_LOGGING_ENABLED"] == 'true',
```

Comment:
Should probably use `dart:core` here. See https://api.flutter.dev/flutter/dart-core/bool/parse.html for more details.

Reply:
done!